### DISCUSSION TITLE
Custom Entity Models (CEM)

### DISCUSSION THREAD

**Discussion by Nhantaa** [IrisShaders/Iris#2373] [Ideas]
Hi 👋 I wanted to know if you plan to add the "Custom Entity Models (CEM)" option from Optifine to Iris because I love this project and I would like the "Fresh Animations" resource pack by FreshLX to work natively and avoid having to do multiple manipulations. I’m suggesting this because I noticed the page "Will (insert feature) part of OptiFine be added to Iris?" and thought of this feature.

I don’t know if it’s possible to implement the CEM from Optifine, but I saw that FreshLX suggests this as an alternative:
https://github.com/Traben-0/Entity_Model_Features
Additionally, in my brief research, I also found this:
https://github.com/YoungSoulluoS/cem_Fork
https://github.com/YoungSoulluoS/Fresh-Animations_cem_Fork
I don’t know if this can help, but in any case, thank you for what you do for the Minecraft community.

**Comment by IMS212** [comment]
Entity Model Features is the officially endorsed way to use CEM. This is not being added to Iris.
